src/rcore.c

```diff
@@ -2558,25 +2536,9 @@ Ray GetMouseRay(Vector2 mouse, Camera camera)
     // Store values in a vector
     Vector3 deviceCoords = { x, y, z };
 
-    // Calculate view matrix from camera look at
-    Matrix matView = MatrixLookAt(camera.position, camera.target, camera.up);
-
-    Matrix matProj = MatrixIdentity();
-
-    if (camera.projection == CAMERA_PERSPECTIVE)
-    {
-        // Calculate projection matrix from perspective
-        matProj = MatrixPerspective(camera.fovy*DEG2RAD, ((double)GetScreenWidth()/(double)GetScreenHeight()), RL_CULL_DISTANCE_NEAR, RL_CULL_DISTANCE_FAR);
-    }
-    else if (camera.projection == CAMERA_ORTHOGRAPHIC)
-    {
-        float aspect = (float)CORE.Window.screen.width/(float)CORE.Window.screen.height;
-        double top = camera.fovy/2.0;
-        double right = top*aspect;
-
-        // Calculate projection matrix from orthographic
-        matProj = MatrixOrtho(-right, right, -top, top, 0.01, 1000.0);
-    }
+    // Get camera matrices
+    Matrix matView = GetCameraViewMatrix(&camera);
```

Comment:
This is a great improvement but camera should be aware at all time of width/height. How is it managed if window is resized?

Reply:
I just noticed from that approach that `rcamera` module is coupled to `rcore` module, current camera module is decouple from other modules, it can be removed at any time without problem (letting the users manage their own cameras).

 Also, moving aspect or width/height to camera side will require a continuous update in case windows was resized or increasing the coupling between modules (doing the camera update in rcore screen resizing functions).